interop-cats/jvm/src/main/scala/zio/interop/catszmanagedjvm.scala

```diff
@@ -118,6 +120,12 @@ sealed trait CatsZManagedInstances1 {
 
 }
 
+sealed trait CatsZManagedInstances2 {
+
+  implicit def monadZManagedInstances[R, E]: Monad[ZManaged[R, E, ?]] = new CatsZManagedMonad
```

Comment:
Doesn't MonadError already include Monad? There are no additional constraints on the MonadError instance, on reading it seems like Monad instance is redundant

Reply:
It was there before so I left it in and just changed the priority.
I'll throw it out. one sec